arch/arm64/boot/dts/qcom/msm8953.dtsi

```diff
@@ -1056,14 +1056,14 @@
 				bias-disable;
 			};
 
-			i2c_6_default: i2c-6-default-state {
+			spi_6_default: spi-6-default-state {
 				pins = "gpio22", "gpio23";
 				function = "blsp_i2c6";
```

Comment:
you need to choose function 1 here, `blsp_spi6`

Also since you find both i2c and spi on these pins, please don't replace the definition